hw/acpi/aml-build.c

```diff
@@ -2308,29 +2308,11 @@ void build_pci_host_bridge(Aml *table, AcpiPciBus *pci_host)
     aml_append(table, scope);
 }
 
-void acpi_dsdt_add_pci_bus(Aml *table, AcpiPciBus *pci_host)
+void build_acpi_pci_hotplug(Aml *scope)
 {
-    Aml *dev;
-    Aml *sb_scope;
-    Aml *scope;
     Aml *field;
     Aml *method;
 
-    sb_scope = aml_scope("_SB");
```

Comment:
Where has this logic moved to? Are we missing a corresponding change to the PIIX code? Does the PIIX PCI hotplug continue to work with these changes?